### ISSUE TITLE
peerDependencies warning for react@16

### ISSUE THREAD

**Issue by afc163** [alibaba/BizCharts#210]
```
peerDependencies WARNING bizcharts-plugin-slider@^2.0.1 requires a peer of react-dom@>15 <16 but react-dom@16.2.0 was installed
peerDependencies WARNING bizcharts-plugin-slider@^2.0.1 requires a peer of react-addons-transition-group@^15.0.0 but none was installed
```

**Comment by afc163** [comment]
ping~

**Comment by weepy3641** [comment]
@afc163 
fix at  bizcharts-plugin-slider@2.0.3
